MatrixSDK/JSONModels/MXJSONModels.m

```diff
@@ -1560,3 +1628,539 @@ - (void)encodeWithCoder:(NSCoder *)aCoder
 }
 
 @end
+
+#pragma mark - Groups (Communities)
+
+@implementation MXGroupProfile
+
++ (id)modelFromJSON:(NSDictionary *)JSONDictionary
+{
+    MXGroupProfile *profile = [[MXGroupProfile alloc] init];
+    if (profile)
+    {
+        NSDictionary *dict = [MXJSONModel removeNullValuesInJSON:JSONDictionary];
```

Comment:
`[MXJSONModel removeNullValuesInJSON:]` should not be required. It was useful for MXEvent at the beginning of the project